src/main/java/org/candlepin/subscriptions/resource/SubscriptionTableController.java

```diff
@@ -269,6 +269,9 @@ private static void sortCapacities(
             case NEXT_EVENT_TYPE:
               diff = left.getNextEventType().compareTo(right.getNextEventType());
               break;
+            case TOTAL_CAPACITY:
```

Comment:
Will this enum be automatically generated in the pipeline or do I need to commit it somewhere? Locally I ran 'openApiGenerate' to test.

Reply:
It's fine. We have the general build task hooked up to openApiGenerate, so that it's sufficient to simply do a `./gradlew assemble` (or any other task that invokes a build).